test/langtools/jdk/javadoc/tool/CheckManPageOptions.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8274211
+ * @summary Test man page that options are documented
+ * @modules jdk.javadoc/jdk.javadoc.internal.tool:+open
+ * @run main CheckManPageOptions
+ */
+
+import jdk.javadoc.doclet.Doclet;
+import jdk.javadoc.doclet.StandardDoclet;
+import jdk.javadoc.internal.tool.ToolOptions;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import java.util.TreeSet;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Checks the set of options found by fuzzy-parsing the troff or Markdown versions
+ * of the javadoc man page against the set of options declared in the source code.
+ */
+public class CheckManPageOptions {
+    public static void main(String... args) throws Exception {
+        new CheckManPageOptions().run(args);
+    }
+
+    static final PrintStream out = System.err;
+
+    // FIXME: JDK-8274295
+    List<String> MISSING_IN_MAN_PAGE = List.of(
+            "--legal-notices",
+            "--link-platform-properties",
+            "--no-platform-links",
+            "--since",
+            "--since-label",
+            "--snippet-path");
```

Comment:
Be specific, even if verbose:
```suggestion
    List<String> MISSING_IN_MAN_PAGE = List.of(
            "--legal-notices",  // FIXME: JDK-8274295
            "--link-platform-properties",  // FIXME: JDK-8274295
            "--no-platform-links",  // FIXME: JDK-8274295
            "--since",  // FIXME: JDK-8274295
            "--since-label",  // FIXME: JDK-8274295
            "--snippet-path"); // FIXME: JDK-8266666
```